src/structures/interfaces/TextBasedChannel.js

```diff
@@ -98,6 +98,14 @@ class TextBasedChannel {
    * @property {string} [append=''] Text to append to every piece except the last
    */
 
+  /**
+   * Options for sending a message with a reply.
+   * @typedef {Object} ReplyOptions
+   * @param {MessageResolvable} messageReference The message to reply to (must be in the same channel and not system)
+   * @param {boolean} [failIfNotExists=true] whether to error if the referenced message
```

Comment:
```suggestion
   * @param {boolean} [failIfNotExists=true] Whether to error if the referenced message
```
jsdoc descriptions of function params start with an uppercase letter everywhere in the lib.

Reply:
🤦 thanks!